README.md

```diff
@@ -1,6 +1,7 @@
 Pa11y Runner
 ===
-`pa11y_runner` is a simple wrapper for running pa11y scans against local copies of websites.
+`pa11y_runner` is a simple wrapper for the [pa11y](https://pa11y.org/) CLI tool for runnign scans against local copies of websites.
```

Comment:
I think it might be nice to clarify what "local copies" means here. Does it mean running this against a web application running on your local machine?

Reply:
Updated